packages/shared/components/ProposalsDetails.svelte

```diff
@@ -30,16 +31,22 @@
             }
         }
     }
+
+    function onAddProposal(): void {
+        openPopup({
+            type: 'addProposal',
+            overflow: true,
+        })
+    }
 </script>
 
 <proposals-details>
-    <header-container class="flex justify-between items-center mb-4">
+    <header-container class="flex justify-left items-center mb-4">
         <Text fontSize="14" fontWeight={FontWeight.semibold}>
             {localize('views.governance.proposalsDetails.title')}
         </Text>
-        <ProposalsDetailsButton />
     </header-container>
-    <ul class="space-y-2">
+    <ul class="space-y-2 mb-4">
```

Comment:
Instead of adding a margin everywhere, it makes more sense to add `space-y-4` to proposals-details.